src/main/scala/gitbucket/core/service/SystemSettingsService.scala

```diff
@@ -1,14 +1,14 @@
 package gitbucket.core.service
 
 import javax.servlet.http.HttpServletRequest
-
 import com.nimbusds.jose.JWSAlgorithm
 import com.nimbusds.oauth2.sdk.auth.Secret
 import com.nimbusds.oauth2.sdk.id.{ClientID, Issuer}
 import gitbucket.core.service.SystemSettingsService._
 import gitbucket.core.util.ConfigUtil._
 import gitbucket.core.util.Directory._
 import gitbucket.core.util.SyntaxSugars._
+import org.eclipse.jetty.http.{HttpHeader, HttpScheme}
```

Comment:
These classes are not available at runtime. It's compilable, but I run GitBucket by `sbt ~jetty:start` or run the executable war file, I got `java.lang.NoClassDefFoundError: org/eclipse/jetty/http/HttpHeader`.

Reply:
fixed